LoopKit/TherapySettings.swift

```diff
@@ -26,15 +26,21 @@ public struct TherapySettings: Equatable {
     
     public var carbRatioSchedule: CarbRatioSchedule?
     
+    public var basalRateSchedule: BasalRateSchedule?
+    
+    public var insulinModel: StoredSettings.InsulinModel?
+    
     public init(
         glucoseTargetRangeSchedule: GlucoseRangeSchedule? = nil,
         preMealTargetRange: DoubleRange? = nil,
-        legacyWorkoutTargetRange: DoubleRange? = nil,
+        workoutTargetRange: DoubleRange? = nil,
         maximumBasalRatePerHour: Double? = nil,
         maximumBolus: Double? = nil,
         suspendThreshold: GlucoseThreshold? = nil,
         insulinSensitivitySchedule: InsulinSensitivitySchedule? = nil,
-        carbRatioSchedule: CarbRatioSchedule? = nil
+        carbRatioSchedule: CarbRatioSchedule? = nil,
+        basalRateSchedule: BasalRateSchedule? = nil,
+        insulinModel: StoredSettings.InsulinModel? = nil
     ){
         self.glucoseTargetRangeSchedule = glucoseTargetRangeSchedule
         self.preMealTargetRange = preMealTargetRange
```

Comment:
I had fixed this in my branch: workoutTargetRange is never initialized. O_o.  (I realize this is somewhat orthogonal to _this_ PR)
```suggestion
        self.preMealTargetRange = preMealTargetRange
        self.workoutTargetRange = workoutTargetRange
```